src/scenario.jl

```diff
@@ -871,15 +872,19 @@ function run_model(
         has_fog_locs::Bool = !isempty(selected_fog_ranks)
 
         # Fog selected locations
-        if fog_decision_years[tstep] && has_fog_locs
+        if has_fog_locs  # fog_decision_years[tstep] &&
```

Comment:
Sorry, this got mixed in by accident.

It is a shift in how the scenarios are conceptualized.
Previously, we were changing how often deployments occur (once every n years, where n may be 1).

Now it seems the emphasis is on deployments occurring every year. So this change allows this, but still allows the selection of deployment locations to change every n years.